### PR TITLE
add terminal plugin

### PR DESCRIPTION
**Pull request by mbkma**
Full credits go to gnome, I just changed some things to the pluma api.
I think a terminal is a very useful plugin, especially when writing latex, where one needs to compile a lot. Other terminal plugins for pluma I found on the internet are very out of date and not easy installable for the end-user.

### CODE REVIEW

File: plugins/terminal/pluma-terminal.metainfo.xml.in [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2014 Igor Gnatenko <i.gnatenko.brain@gmail.com> -->
+<component type="addon">
+  <id>pluma-terminal</id>
+  <extends>org.pluma.mate.desktop</extends>
+  <name>Terminal</name>
+  <summary>A simple terminal widget accessible from the bottom panel</summary>
+  <url type="homepage">https://wiki.gnome.org/Apps/Gedit/ShippedPlugins</url>
+  <url type="bugtracker">https://gitlab.gnome.org/GNOME/gedit-plugins/issues</url>
```
Comment (sc0w) [inline plugins/terminal/pluma-terminal.metainfo.xml.in:9]: I suppose we have here a forked version, I think it makes sense to have mate url instead

File: plugins/terminal/terminal.plugin.desktop.in [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+Loader=python3
+Module=terminal
+IAge=3
+Name=Embedded Terminal
+Description=Embed a terminal in the bottom pane.
+# Translators: Do NOT translate or transliterate this text (this is an icon file name)!
+Icon=utilities-terminal
+Authors=Paolo Borelli
+Copyright=Copyright © 2005 Paolo Borelli
+Website=http://www.gedit.org
```
Comment (sc0w) [inline plugins/terminal/terminal.plugin.desktop.in:11]: I suppose we have here a forked version, I think it makes sense to have mate url instead

**Comment by raveit65** [comment]
@mate-desktop/core-team 
Maybe, its time to start with an extra package pluma-plugins ?

**Comment by sc0w** [comment]
@raveit65  I agree, I have no problem if someone like to work in that

**Comment by raveit65** [comment]
@mbkma 
Do you plan to add more plugins to pluma?
Or is this the last one?

**Comment by mbkma** [comment]
I do not know yet. In October I will have some free time, maybe then I can port more plugins to pluma.

**Comment by raveit65** [comment]
My question is more if we should start with extra package pluma-plugins?
Or do you prefer to split it later when someone really add new plugins?

**Comment by mbkma** [comment]
@raveit65 I prefer to split it later when someone really add new plugins. The terminal-plugin then can be a part of the new pluma-plugin package. If you agree, you can close this PR.

**Comment by raveit65** [comment]
Ok, we can just leave it open a "sleeper" PR.

**Comment by mbkma** [comment]
Until now forked 3 gedit plugins: codecomment, synctex and terminal. I created a package which you can include in mate if you want: https://github.com/mbkma/pluma-plugins. For more complex plugins, more changes to pluma are necessary which I didn't manage to implement yet.

**Comment by raveit65** [comment]
Nice, i can include it to  Mate. Do you like to be become maintainer for pluma-plugins and pluma?

**Comment by mbkma** [comment]
It would be a honour for me :)

**Comment by mbkma** [comment]
@raveit but dont include it yet I still have to test some things

**Comment by raveit65** [comment]
Too late, it's already done.
I send you an invitation and you can work now directly at https://github.com/mate-desktop/pluma-plugins.
No PR needed for first adjustments.
Or should i delete the repo?

**Comment by mbkma** [comment]
it doesn't build right now. But I think I can fix it within the next hours. So if that is no problem, you don't have to delete the repo.

**Comment by mbkma** [comment]
See https://github.com/mate-desktop/pluma-plugins/pull/1. Maybe when this PR is merged you can create your own repository pluma-plugins such that it is not forked from mine (looks nicer if it is directly from MATE).

**Comment by raveit65** [comment]
I think renaming/renaming-back should decouple it from your repo

**Comment by mbkma** [comment]
The terminal plugin is part of the new package `pluma-plugins`.

**Pull request closed without merging**